gen/sshkeys

```diff
@@ -7,34 +7,26 @@ use File::Basename;
 
 our $SERVICE_NAME = basename($0);
 our $PROTOCOL_VERSION = "3.1.0";
-my $SCRIPT_VERSION = "3.0.1";
+my $SCRIPT_VERSION = "3.0.2";
 
 perunServicesInit::init;
 my $DIRECTORY = perunServicesInit::getDirectory;
-my $data = perunServicesInit::getHierarchicalData;
+my $data = perunServicesInit::getHashedHierarchicalData;
 
 my $sshkeysDirectory = "$DIRECTORY/sshkeys";
 mkdir $sshkeysDirectory or die "Can't mkdir $sshkeysDirectory: $!";
 
 #Constants
 our $A_SSHKEYS;                      *A_SSHKEYS =                     \'urn:perun:user:attribute-def:def:sshPublicKey';
-our $A_USER_LOGIN;                   *A_USER_LOGIN =                  \'urn:perun:user_facility:attribute-def:virt:login';
-
-my $service_file_name = "$DIRECTORY/$::SERVICE_NAME";
+our $A_USER_FACILITY_LOGIN;          *A_USER_FACILITY_LOGIN =         \'urn:perun:user_facility:attribute-def:virt:login';
 
 my %sshKeys;
 
-my %facilityAttributes = attributesToHash $data->getAttributes;
-#####################################
-
-my @resourcesData = $data->getChildElements;
-foreach my $rData (@resourcesData) {
-	my %values;
-	my @membersData = $rData->getChildElements;
-	foreach my $mData (@membersData) {
-		my %memberAttributes = attributesToHash $mData->getAttributes;
-
-		$sshKeys{$memberAttributes{$A_USER_LOGIN}} = $memberAttributes{$A_SSHKEYS};
+foreach my $resourceId ($data->getResourceIds()) {
```

Comment:
These two for each cycles can be merged into one by using the method `getMemberIdsForFacility()`. Sorry that I did not mention it earlier. I forgot that we have this method.